src/pages/ContentScripts/features/repo-collection/view.tsx

```diff
@@ -0,0 +1,346 @@
+import React, { useEffect, useState } from 'react';
+import { DownOutlined } from '@ant-design/icons';
+import { Dropdown, Space, Divider, Button, Input, theme } from 'antd';
+import type { MenuProps } from 'antd';
+import {
+  AppstoreOutlined,
+  GithubOutlined,
+  SettingOutlined,
+  DeleteOutlined,
+  FolderAddOutlined,
+} from '@ant-design/icons';
+import { getRepoName } from '../../../../helpers/get-repo-info';
+
+const iconMap: any = {
+  AppstoreOutlined: AppstoreOutlined,
+  GithubOutlined: GithubOutlined,
+  SettingOutlined: SettingOutlined,
+  DeleteOutlined: DeleteOutlined,
+  FolderAddOutlined: FolderAddOutlined,
+};
+
+const { useToken } = theme;
+
+type MenuItem = Required<MenuProps>['items'][number];
+
+function getItem(
+  label: React.ReactNode,
+  key: React.Key,
+  icon?: React.ReactNode,
+  children?: MenuItem[],
+  type?: 'group',
+  disabled?: Boolean
+): MenuItem {
+  return {
+    key,
+    icon,
+    children,
+    label,
+    disabled,
+    type,
+  } as MenuItem;
+}
+
+const View = () => {
+  const { token } = useToken();
+  const contentStyle: React.CSSProperties = {
+    backgroundColor: token.colorBgElevated,
+    borderRadius: token.borderRadiusLG,
+    boxShadow: token.boxShadowSecondary,
+  };
+  const menuStyle: React.CSSProperties = {
+    boxShadow: 'none',
+  };
+  const DefaultItems = [
+    getItem('X-Lab', 'collection1', <GithubOutlined />, [
+      getItem('delete collection', 'delete1', <DeleteOutlined />),
+      getItem('add current Repo', 'add1', <FolderAddOutlined />),
+      { type: 'divider' },
+      getItem('open-digger', '1.1'),
+      getItem('open-perf', '1.2'),
+      getItem('open-leaderboard', '1.3'),
+      getItem('open-wonderland', '1.4'),
+    ]),
+    getItem('Hypertrons', 'collection2', <AppstoreOutlined />, [
+      getItem('delete collection', 'delete2', <DeleteOutlined />),
+      getItem('add current Repo', 'add2', <FolderAddOutlined />),
+      { type: 'divider' },
+      getItem('hypertrons-crx', '2.1'),
+      getItem('hypertrons-crx-website', '2.2'),
+    ]),
+    getItem('Collection Three', 'collection3', <SettingOutlined />, [
+      getItem('delete collection', 'delete3', <DeleteOutlined />),
+      getItem('add current Repo', 'add3', <FolderAddOutlined />),
+      { type: 'divider' },
+      getItem('Option 9', '3.1'),
+      getItem('Option 10', '3.2'),
+      getItem('Option 11', '3.3'),
+      getItem('Option 12', '3.4'),
+    ]),
+  ];
+  const [items, setitems] = useState(DefaultItems);
+  const [refresh, setRefresh] = useState(0);
+  const [newCollectionName, setNewCollectionName] = useState('');
+  const [open, setOpen] = useState(false);
+
+  //将 Repo Collection 信息转化为 `JSON` 用于存至 Chrome 本地
```

Comment:
Hmm, pretty much translation debts are left in commits, as I can see. But it's OK, don't forget it in the final push.